src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.cs

```diff
@@ -69,7 +69,6 @@ public enum CreateObjectFlags
     /// <summary>
     /// Class for managing wrappers of COM IUnknown types.
     /// </summary>
-    [SupportedOSPlatform("windows")]
```

Comment:
Realized this it is not actually supported on all platforms, since it is coreclr only. We probably want to have `UnsupportedOSPlatform` for android, browser, ios, etc.

Reply:
I'm not sure how scalable that is. I'm reaching out to the .NET API Review team for guidance here.

Reply:
Done.